.github/workflows/publish.yaml

```
@@ -0,0 +1,42 @@
+name: checks
+on:
+  pull_request:
+  push:
+    branches:
+      - 'master'
+      - 'main'
+    tags:
+      - 'v*'
+jobs:
+
+  image:
+    runs-on: ubuntu-latest
+    steps:
+      -
+        name: Login to image registry
+        uses: docker/login-action@v1
+        with:
+          registry: ${{ env.image-registry }}
+          username: ${{ secrets.quay_username }}
+          password: ${{ secrets.quay_password }}
```

```suggestion
          username: ${{ secrets.QUAY_USERNAME }}
          password: ${{ secrets.QUAY_PASSWORD }}
```

GitHub automatically capitalised the secret names when I added them, so just to be sure, we should capitalise them here as well.